server/http_hooks.go

```diff
@@ -45,3 +51,66 @@ func (p *Plugin) handleStatus(w http.ResponseWriter, r *http.Request) {
 func (p *Plugin) handleHello(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("Hello World!"))
 }
+
+func (p *Plugin) handleEphemeralUpdate(w http.ResponseWriter, r *http.Request) {
+	request := model.PostActionIntegrationRequestFromJson(r.Body)
+	URL := fmt.Sprintf("%s", *p.API.GetConfig().ServiceSettings.SiteURL)
+
+	count := int(request.Context["count"].(float64)) + 1
+
+	post := &model.Post{
+		Id:        request.PostId,
+		ChannelId: request.ChannelId,
+		UserId:    request.UserId,
```

Comment:
Same here and with `ChannelId`.

Reply:
Without `ChannelId` , UpdateEphemeralPost doesn't appear to work for my tests.

Reply:
That's true, I ran into the same thing. Worth a ticket, or by design?

Reply:
As ChannelId is usually readily available, I am 0/5 if it stays in.  That being said, it seems PostId may be enough by itself?

Reply:
After thinking a bit more about this, I would leave it as it is. The only require informations are `Id` and `ChannelId`, correct? That seams fine to me.